qiskit/providers/aer/primitives/__init__.py

```diff
@@ -0,0 +1,35 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+===================================================
+Primitives (:mod:`qiskit.providers.aer.primitives`)
```

Comment:
It may indeed be possible to make them common, but it is difficult at this point because it is not obvious whether they can be made common to `Discriminator` and `Integrator` in the future. We might do it in the future code refactoring.